app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java

```diff
@@ -200,10 +200,7 @@ public void unCheckAll() {
      * Sets the action bar title to app name when all files have been unchecked
      */
     private void updateActionBarTitle() {
-        ActionBar actionBar = ((AppCompatActivity) mActivity).getSupportActionBar();
-        if (actionBar != null) {
-            actionBar.setTitle(R.string.app_name);
-        }
+        mActivity.setTitle(R.string.app_name);
     }
```

Comment:
The fix works fine for me. @Swati4star Just check if those 4 removed lines are required for some other bug fix. I remember pointing you out something like this. Otherwise, it seems fine.